platforms/native/engine/jsb-gfx.js

```diff
@@ -57,8 +57,7 @@ deviceProto.copyTexImagesToTexture = function (texImages, texture, regions) {
 
 const oldDeviceCreateSwapchainFunc = deviceProto.createSwapchain;
 deviceProto.createSwapchain = function (info) {
-    // In openharmony, we need to get the window handle through the jsb interface
-    info.windowHandle = window.oh ? jsb.device.getWindowHandle() : jsbWindow.windowHandler;
+    info.windowHandle = jsbWindow.windowHandle;
```

Comment:
can we remove this interface for now ? @qiuguohua 
`jsb.device.getWindowHandle() `

This interface is only for OH platform, should also remove the js binding

Reply:
Removed at : https://github.com/cocos/cocos-engine/pull/16093/commits/44f999bcae27ccc36e5b87ec7f26ccc806bf1e38

@qiuguohua Plz review that.